osm2streets-java/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+javac StreetNetwork.java
+cargo build
+java -Djava.library.path=../target/x86_64-unknown-linux-gnu/debug/ StreetNetwork
```

Comment:
You probably need to change this on you end; it's likely just `../target/debug`. I have the architecture bit in there so compiling native and wasm32 doesn't stomp on the build directories